lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/cc/Parser.java

```diff
@@ -131,17 +132,26 @@ public class Parser extends AbstractParser {
   
   public void
   parse(DefaultHandler the_handler) throws FileNotFoundException, ParseException {
+    Element canvas_module_meta = null;
+    try {
+	  canvas_module_meta = this.getXML(utils, CANVAS_MODULE_META);
+	  log.debug("Found - {} {}", CANVAS_MODULE_META, canvas_module_meta);
+    } catch (Exception e) {
+		log.debug("{} not found " + e, CANVAS_MODULE_META);
```

Comment:
Spacing

Reply:
The spacing on these lines includes a mix of tabs and spaces - sigh.   I will fix the spacing by matching the current spacing.